### PR TITLE
Ddp 4378 normalize activity name

### PR DESCRIPTION
**Pull request by yufengwng**
## Context and the big picture

Renamed
* `name` -> `title`
* `dashboardName` -> `name`

Will also fixes DDP-4376.

 ## What type of change is this?
  
  - [x] Bugfix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] Experimental proof-of-concept
  - [ ] Infosec/Compliance remediation
  - [x] Refactor
  
 ## Risk Assessment
  - [ ] These changes are **HIGH RISK**.
    - [ ] There is an elevated risk that may impact a large number of features
    - [ ] There is an elevated risk to security and privacy
    - [ ] These changes introduce new 3rd party dependencies, alter the network perimeter, or contain
    major upgrades to persistent storage or SaaS dependencies such as auth0, sendgrid, or easypost    
  - [x] These changes are low risk, do not impact security/privacy, do not impact pepper shared
  components/modules, and have no major upgrades to 3rd party services or libraries
 
 ### FUD Score 
 Overall, how are you feeling about these changes?
  - [x] :relaxed: All good, business as usual!
  - [ ] :sweat_smile: There might be some issues here
  - [ ] :scream:I'm sweaty and nervous
 
 ## How to we demo these changes?
 How does one observe these changes in a deployed system?  Note that _user visible_ encompasses
 many personas--not just patients and study staff, but ops your fellow devs, compliance, etc.
 
 - [x] They are user-visible in dev as a regular user journey and require no additional instructions.
 - [ ] Getting dev into a state where this is user-visible requires some tech fiddling.  I have
 documented these steps in the related ticket.
 - [ ] Requires other features before it's human visible.  I have documented the blocking issues
 in jira.
 - [ ] I have no idea how to demo this.  Please help me!
 
 ## UI/UX
  - [x] There ain't no UI/UX in these changes
  - [ ] My changes have passed muster with Erin and Jen via an over-the-shoulder review, screenshots, etc.
  - [ ] Erin and Jen have not had an opportunity to provide feedback yet
 
###  Backend
   - [ ] I have considered error handling and notification to slack alert rooms via `LOG.error()`.
     #### Route Changes
     - [ ] My changes involve a new route changes to an existing route, so the who/what/when are logged using the usual logback mechanism
at the _start_ of the route, prior to any validation, as well as at the end of the route when work completes.
     ### Housekeeping Changes
     - [ ] My changes involve updates to housekeeping, and I am logging the who/what/when via logback.
  ### Client-side
   - [ ] My changes do not involve backend route change, but I have useful logging statements nonetheless.

## Analytics
 - [ ] I have discussed the analytics needs at both a platform and study level with Jen and instrumented code
 accordingly.
 - [x] There are no analytics requirements for these changes

## Security and Privacy
Ensuring proper security and privacy for our users is essential.

  ###  Backend
   - [ ] These changes alter fundamental auth filter logic
   - [ ] These changes introduce a new backend API route that is behind an auth filter
   - [ ] These changes introduce a new backend API that _does not require auth_
   - [ ] These changes alter auth0 internals (rules, hosted login, configuration, etc.)
   - [ ] These changes expose new information to elastic and I have verified that PHI is not being exposed to the public
    
  ### Browser
  - [ ] These changes alter what we store in browser storage
  
  ### Mobile
  - [ ] TBD 
 
 ## Performance and Stability
  - [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, and performance bottlenecks and written a brief summary in this PR
  - [ ] I'm unsure about stability, performance, fault tolerance, and graceful degradation.  Please help!
  
## Testing
 - [ ] I have written automated positive tests
 - [ ] I have written automated negative tests
 - [x] I have written zero automated tests but have poked around locally to verify proper functionality
 - [ ] The jira ticket has acceptance criteria and Kiara has in the information she needs to test my changes
 
## Release
 - [x] These changes require no special release procedures--just code!
 - [ ] Releasing these changes requires special handling
   - [ ] I have documented the special procedures in the release plan document
   - [ ] The special procedures require minimal downtime
   - [ ] The special procedures require unavoidable, extended downtime for:
     - [ ] Participant UX
     - [ ] Study staff

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/json/activity/ActivityInstanceSummary.java [hunk 1]
```diff
@@ -11,20 +11,22 @@
     @SerializedName("instanceGuid")
     private String activityInstanceGuid;
 
-    @SerializedName("activityName")
+    // todo: rename to `activityName`
+    @SerializedName("activityDashboardName")
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/json/activity/ActivityInstanceSummary.java:15]: Changing it in this PR will change the API. I'll do the change in a separate PR instead.

File: pepper-apis/src/main/resources/db-changes/schema/DDP-4378-normalize-activity-name.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                   on dashboard.study_activity_id = title.study_activity_id
+                   and dashboard.language_code_id = title.language_code_id
+        </sql>
+    </changeSet>
+
+    <changeSet author="yufeng" id="20200204-cleanup-activity-i18n-tables">
+        <dropTable tableName="i18n_study_activity_name_trans"/>
+        <dropTable tableName="i18n_study_activity_subtitle_trans"/>
+        <dropTable tableName="i18n_study_activity_description_trans"/>
+        <dropTable tableName="i18n_study_activity_dashboard_name_trans"/>
```
Comment (yufengwng) [inline pepper-apis/src/main/resources/db-changes/schema/DDP-4378-normalize-activity-name.xml:66]: Consolidated into new `i18n_study_activity` table. CC @MocanaAtBroad

File: study-builder/studies/angio/about-you.conf [hunk 3]
```diff
@@ -14,16 +14,16 @@
   "translatedNames": [
     {
       "language": "en",
-      "text": "Join the movement: tell us about yourself"
+      "text": "Initial Enrollment Survey"
     }
   ],
-  "translatedSubtitles": [],
-  "translatedDashboardNames": [
+  "translatedTitles": [
```
Comment (yufengwng) [inline study-builder/studies/angio/about-you.conf:20]: @charlottedye you'll need to update your Prion config files, similar to what I have done here for the other studies.

**Comment by yufengwng** [comment]
please run tests

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Comment by yufengwng** [comment]
Merging this into `develop` now so @charlottedye can get a head-start on updating Prion.

**Pull request merged**